docs/01_getting-started/03_go-to-the-next-level/06_forms.md

```diff
@@ -19,17 +19,17 @@ To interact with the Event Handler that you created [previously](../../../gettin
 Start with the Form component, which will generate all the inputs based on the API:
 
 ```html title='home.template.ts'
-<zero-form
+<foundation-form
   resourceName="EVENT_TRADE_INSERT"
-></zero-form>
+></foundation-form>
 ```
 
 To respond to the user clicking on the **Submit** button, you need to call the `EVENT_TRADE_INSERT` event:
 ```html {3} title='home.template.ts'
-<zero-form
+<foundation-form
   resourceName="EVENT_TRADE_INSERT"
   @submit=${(x, c) => x.insertTrade(c.event as CustomEvent)}
-></zero-form>
+></foundation-form>
 ```
```

Comment:
When it is loaded to the web page, it's not the same as the others. Is that correct? 
![image](https://user-images.githubusercontent.com/120386099/224333119-fb8881d3-a4e2-4e9e-ab58-547d8bcbb9e1.png)

Reply:
Hi @GuiWar can you elaborate on that, I am not quite sure what you mean sorry?

Reply:
Do you mean the syntax highlighting?

Reply:
If it is, it is because the syntax highlighting is a bit stuck here because we're using a mix of html and typescript interpolation